### ISSUE TITLE
Assinged issue did not move "Project already has the associated issue"

### ISSUE THREAD

**Issue by CiTRuS93** [alex-page/github-project-automation-plus#28]
hi there!

I have one project with three column and one issue on the Todo column.
i want to move the issue when i assign it to some one.

my workflow:
```name: Move assigned issues into InProgress

on:
 issues:
    types: [assigned]

jobs:
  automate-project-columns:
    runs-on: ubuntu-latest
    steps:
      - uses: alex-page/github-project-automation-plus@v0.1.3
        with:
          project: presentaion project
          column: In progress
          repo-token: ${{ secrets.GH_PAT }}
```

but i'm keep getting this error:

```Run alex-page/github-project-automation-plus@v0.1.3
  with:
    project: presentaion project
    column: In progress
    repo-token: ***
##[error]Project already has the associated issue
```

**Comment by alex-page** [comment]
Is the project open source? Can I have a look how it is failing in an issue?

**Comment by CiTRuS93** [comment]
i'll add you as a collaborator, i'm just trying diffrent thing in git hub

**Comment by alex-page** [comment]
I am looking into this now. Thanks @CiTRuS93.

**Comment by alex-page** [comment]
This might be related, diving into it now:
https://github.com/alex-page/github-project-automation-plus/pull/29

**Comment by alex-page** [comment]
@CiTRuS93 can you please try `v.0.2.0`?

**Comment by CiTRuS93** [comment]
Still the same error

**Comment by alex-page** [comment]
Thanks mate I am looking into it 👀

**Comment by alex-page** [comment]
This should be resolved in `v0.2.1` let me know if you have any problems. Happy to reopen the issue. Thanks again @CiTRuS93!

**Comment by CiTRuS93** [comment]
Working perfectly good! Thanks!
